README.md

```diff
@@ -77,6 +77,24 @@ Cloud Storage for your project.
 See the [Storage client library docs][storage-client-lib-docs] to learn how to interact
 with the Cloud Storage using this Client Library.
 
+Alternative: NIO
+----------------
+
+We also provide a separate library,
+[google-cloud-nio](https://github.com/googleapis/java-storage-nio)
+(currently in alpha). It lets you create standard Java `Path` objects that can
```

Comment:
I don't think we should mention or recommend alpha products here,.

Reply:
All right, if you don't want it mentioned I will close this pull request.